### PR TITLE
Normalize 1 -> t

### PR DESCRIPTION
**Pull request by ar1a**
Cleanup from old code? Annoyed me as I was scrolling through stuff.

### CODE REVIEW

File: core/core-editor.el [hunk 1]
```diff
@@ -129,7 +129,11 @@ savehist file."
 
 (def-package! recentf
   ;; Keep track of recently opened files
+<<<<<<< HEAD
```
Comment (ar1a) [inline core/core-editor.el]: sigh

**Comment by hlissner** [comment]
These were intentional. `:defer 1` instructs the package to autoload after 1 second of idle time, but perhaps the new `:defer-incrementally` would be better, especially for helm.

**Comment by ar1a** [comment]
ah, will org get that done to it as well?

**Comment by hlissner** [comment]
[It already has!](https://github.com/hlissner/doom-emacs/blob/develop/modules/lang/org/config.el#L28-L31) Same with [magit](https://github.com/hlissner/doom-emacs/blob/develop/modules/tools/magit/config.el#L19).

**Pull request closed without merging**